types/node/process.d.ts

```diff
@@ -1044,6 +1044,21 @@ declare module "process" {
                  * @param [signal='SIGTERM'] The signal to send, either as a string or number.
                  */
                 kill(pid: number, signal?: string | number): true;
+                /**
+                 * Loads the environment configuration from a `.env` file into `process.env`. If
+                 * the file is not found, error will be thrown.
+                 *
+                 * To load a specific .env file by specifying its path, use the following code:
+                 *
+                 * ```js
+                 * import { loadEnvFile } from 'node:process';
+                 *
+                 * process.loadEnvFile('./development.env')
+                 * ```
+                 * @since v21.7.0
+                 * @param path The path to the .env file
+                 */
+                loadEnvFile(path: string | URL | Buffer | undefined): void;
```

Comment:
`loadEnvFile(path?:` - by the doc we can call it without argument - `loadEnvFile()`

Reply:
resolved in: 99a90c3aaaa7bb7390725f885041890bcaf05369